Marlin/src/core/boards.h

```diff
@@ -131,6 +131,7 @@
 #define BOARD_PXMALION_CORE_I3        1164  // Pxmalion Core I3
 #define BOARD_PANOWIN_CUTLASS         1165  // Panowin Cutlass (as found in the Panowin F1)
 #define BOARD_KODAMA_BARDO            1166  // Kodama Bardo V1.x (as found in the Kodama Trinus)
+#define BOARD_DAGOMA_D6               1167  // Dagoma D6
```

Comment:
It'd be helpful to note which machine this board is from like the comments above it. Like so: 

`// Dagoma D6 (as found in the DiscoUltimate V2 TMC)`